cohort-engine/src/main/java/com/ibm/cohort/engine/cos/CosConfiguration.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * (C) Copyright IBM Corp. 2021, 2021
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.cohort.engine.cos;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.beust.jcommander.DynamicParameter;
+import com.beust.jcommander.Parameter;
+
+public class CosConfiguration {
```

Comment:
Per my earlier comment, I'm not sure if COS specific information has a place here.
That does mean tricky things for how we handle configuration of the COS information.
We may want to use a combination of config files (like Caffeine) and environment variables instead.